src/site/stages/build/drupal/graphql/vetCenter.graphql.js

```diff
@@ -63,10 +86,9 @@ const vetCenterFragment = `
 const GetVetCenters = `
   ${vetCenterFragment}
   
-  query GetVetCenters($onlyPublishedContent: Boolean!) {
```

Comment:
We should not check in this change!

Maybe we can come up with a way to toggle this using a command-line switch in the build?

Reply:
ok